CHANGELOG.md

```diff
@@ -2,8 +2,21 @@
 All notable changes to this project will be documented in this file.
 
 ## 13.0.0-rc.1 – 2021-11-11
+### Added
+- Moderators can now set permissions for all or individual users to control if they can enable audio and video, do a screenshare, start a call and ignore the lobby.
+- When starting a call is limited to moderators, they can now also end the call for all participants.
+- A device checker makes sure users start with the desired microphone and camera and also allows them to disable the devices before joining a call.
+- Users can now blur their background (Not supported on Safari)
+- Users can not opt-out of call notifications on a conversation basis
+
+### Changed
+- The layout of the call view has been redesigned
+- The size of the grid view is now limited to 20 videos by default to reduce the performance impact.
+- Version 0.4.0 of the High-performance backend is now required
```

Comment:
`of the signaling server of the High-performance backend`, as the version of the meta packages of the HPB is 0.3.4, only the signaling server is 0.4.